src/main/java/frc/robot/subsystems/Shooter.java

```diff
@@ -34,14 +38,11 @@ public class Shooter extends SubsystemBase {
   private final String SMART_DASHBOARD_POSITION = "Shooter Motor Position";
   private final String SMART_DASHBOARD_TARGET_POSITION = "Shooter Motor Target Position";
 
-  private final CANcoder armPositionEncoder = new CANcoder(18);
-  private double targetArmPosition = 0;
+  private final CANcoder armPositionEncoder = new CANcoder(18, Constants.OperatorConstants.canivoreSerial);
+  private double targetArmPosition = 40;
```

Comment:
possibly comment here that it is the default arm position for the beginning of the match.